src/main/java/spring/impls/Administrator.java

```diff
@@ -0,0 +1,32 @@
+package spring.impls;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Log4j
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Administrator implements Lyricist {
```

Comment:
Why not add a constructor with parameters? And initialize an object of this class using not default constructor but constructor with param.